docs/frontend/javascript-approaches/initializers.md

```diff
@@ -4,46 +4,10 @@ sidebar_position: 1
 
 # Initializers & the asset pipeline
 
-> app/assets/javascripts/initializers
-
 :::important
 
 When refactoring or adding new functionality that lives in the asset pipeline, strongly consider moving it to Packs and Webpacker.
 
 :::
 
-The asset pipeline is where we have the "legacy" JavaScript. It's legacy in the sense that it's not EcmaScript Modules (ESM). It's a bunch of JS files concatenated together being served using [Sprockets](https://github.com/rails/sprockets-rails) which packages static assets. They are available globally.
-
-This source code is not transpiled, only packaged and minified, and will be limited to whatever flavor of JavaScript can run on the user's web browser.
-
-`app/assets/javascripts/application.js` contains the manifest JavaScript file which is included globally in the primary template, `app/views/layouts/application.html.erb`.
-
-`application.js` automatically includes all JS files via the statement:
-
-```
-//= require_tree .
-```
-
-One JS file in particular, `app/assets/javascripts/initializePage.js`, bootstraps the majority of the functionality and calls its initializers on every page. You will notice, within this file, that major sections of the websites are bootstrapped, for example:
-
-```
-initializeBaseTracking();
-initializeCommentsPage();
-initEditorResize();
-initLeaveEditorWarning();
-initializeArticleReactions();
-initNotifications();
-initializeSplitTestTracking();
-```
-
-All the "initializers" are in `/app/assets/javascripts/initializers`.
```

Comment:
I have a quick question that I may have missed somewhere - apologies if I did:

In `app/assets/javascripts/initializers` we still have a few files that have not been migrated. 

<img width="480" alt="Screenshot 2023-05-11 at 11 07 51" src="https://github.com/forem/forem-docs/assets/2786819/7b506bd1-1569-47b9-a047-405c2adee21c">

Are we still planning on tackling them or will they continue to live in the initializers? If there is a reason why we're not migrating these over, perhaps we should still keep some of this section and leave a reason why.

Reply:
@lboogie04 please let me know your thoughts on this so we can plan accordingly for this initiative

Reply:
> I would still keep the documentation the same because we wouldn't want code updated and/or created in asset pipeline but rather encourage contributors to only use the app/javascript directory.

I agree with this however I also feel like if we completely remove this section of documentation without referencing it anywhere else or putting a more brief explanation to describe why we still have files in `app/assets/javascripts/initializers`  it will confuse folks (maybe a newer employee or an OSS contributor) who are looking at `app/assets/javascripts/initializers` in the application and wondering where this fits into the bigger scheme of things. We're also not certain how long it will take to migrate these additional files. Hence, I think we still need some reference to this until we completely remove everything - maybe we strongly emphasize that folks should use the app/javascript directory and outline that PR's that continue adding to this section will be rejected or something similar.